examples/mqtt_winc1500/mqtt_winc1500.ino

```diff
@@ -60,18 +58,15 @@ Adafruit_MQTT_Subscribe onoffbutton = Adafruit_MQTT_Subscribe(&mqtt, AIO_USERNAM
 
 
 void setup() {
-#ifdef WINC_EN
-  pinMode(WINC_EN, OUTPUT);
-  digitalWrite(WINC_EN, HIGH);
-#endif
```

Comment:
I tested this with [Adafruit Feather M0 WiFi - ATSAMD21 + ATWINC1500](https://www.adafruit.com/products/3010) and it worked without setting the WINC_EN pin. I didn't have a [Adafruit ATWINC1500 WiFi Breakout](https://www.adafruit.com/products/2999) to test with but looking at the WiFi101 code I don't think this is needed since it sets the pin mode for it.